### ISSUE TITLE
    Problem in some data (string with a comma (,) rather than a period (.)) in float() transformation

### ISSUE THREAD

**Issue by carloborsoi** [jjCode01/xer-reader#1]
In Portuguese, floating-point numbers are denoted with a comma (,) rather than a period (.). This means that while in English a floating-point number might be written as 3.14, in Portuguese it would be written as 3,14.

This difference can sometimes lead to errors for some P6 projects during float() transformation from string ( ex. valiators.py line 24).

**Comment by jjCode01** [comment]
Thank you for letting me know, I didn't think of this. Float conversion is only used in one place in the `xer-reader` project you posted this issue to.  I've corrected the issue by adding `.replace(",", ".")` to the str being transformed to a float. 

This issue has also been corrected in the `xerparser` project, which has the validator.py file you referenced.

Thank you again, I really appreciate the your input!
